lib/modules/manager/asdf/upgradeable-tooling.ts

```diff
@@ -332,6 +332,14 @@ export const upgradeableTooling: Record<string, ToolingDefinition> = {
       extractVersion: '^v(?<version>\\S+)',
     },
   },
+  maven: {
+    asdfPluginUrl: 'https://github.com/halcyon/asdf-maven.git',
+    config: {
+      datasource: GithubReleasesDatasource.id,
```

Comment:
```suggestion
      datasource: MavenDatasource.id,
```

also needs an:
`import { MavenDatasource } from '../../datasource/maven';`